test/suites/common-tanssi/fees/test_fee_balance_transfer.ts

```diff
@@ -91,13 +91,35 @@ describeSuite({
                     signedTx.encodedLength
                 );
 
+                const feeMultiplier = (await polkadotJs.query.transactionPayment.nextFeeMultiplier()).toBigInt();
```

Comment:
I am wondering whether we should move this to common instead of common-tanssi. I guess all runtimes should behave equally with respect to this. @ParthDesai can you try it?

Reply:
thank you!